integrations/slack/src/middlewares.ts

```diff
@@ -34,12 +44,166 @@ export async function verifySlackRequest(request: Request, { environment }: Slac
     }
 }
 
+const acknowledgeQuery = async ({
```

Comment:
1. Let's use the `function` declaration syntax
2. We can remove the async here since we're not awaiting?